src/wallet/wallet2.cpp

```diff
@@ -6869,13 +6873,17 @@ std::vector<wallet2::pending_tx> wallet2::create_transactions_2(std::vector<cryp
 				++original_output_index;
 			}
 
-			if(available_amount > 0 && !dsts.empty() && estimate_rct_tx_size(tx.selected_transfers.size(), fake_outs_count, tx.dsts.size(), bulletproof) < TX_SIZE_TARGET(upper_transaction_size_limit))
+			if(available_amount > 0 && !dsts.empty() && 
+				estimate_rct_tx_size(tx.selected_transfers.size(), fake_outs_count, tx.dsts.size(), bulletproof) < TX_SIZE_TARGET(upper_transaction_size_limit))
 			{
-				// we can partially fill that destination
-				LOG_PRINT_L2("We can partially pay " << get_public_address_as_str(m_nettype, dsts[0].is_subaddress, dsts[0].addr) << " for " << print_money(available_amount) << "/" << print_money(dsts[0].amount));
-				tx.add(dsts[0].addr, dsts[0].is_subaddress, available_amount, original_output_index, m_merge_destinations);
-				dsts[0].amount -= available_amount;
-				available_amount = 0;
+				if(!bulletproof || tx.dsts.size() < cryptonote::common_config::BULLETPROOF_MAX_OUTPUTS-1)
```

Comment:
Must this be `<=` because we can have 15 outputs.

Reply:
Same.